### DISCUSSION TITLE
[calibration_source -> camera: ] is it a string or a boolean?

### DISCUSSION THREAD

**Discussion by gsotiriou** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#192] [Q&A] [answered]
In the documentation (https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card#calibration-source-options) it states that for `calibration_source` the `camera:` dictionary value is a `string`.

In the demo example (https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/blob/master/docs/demo_config.yaml#L8) it is indicated that it is a `boolean` (from the value `true`). 

Which is correct? Tried with both and it "seems to work" with both which is not particularly reasonable.

**Comment by PiotrMachowski** [comment, marked as answer]
This is a mistake in documentation, thank you for pointing it out!

**Reply by PiotrMachowski** [reply]
Fixed

**Reply by gsotiriou** [reply]
Thank you!
